src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -323,7 +323,8 @@ private void ReportSessionWideTelemetry()
 
         private void DisposeVisualStudioServices()
         {
-            _workspace?.Services.GetRequiredService<VisualStudioMetadataReferenceManager>().DisconnectFromVisualStudioNativeServices();
```

Comment:
VisualStudioMetadataReferenceManager is already a normal MEF service.  so no need for a workspace to get it.

Reply:
Nice!